go-controller/pkg/libovsdb/ops/model.go

```diff
@@ -489,15 +490,40 @@ func buildFailOnDuplicateOps(c client.Client, m model.Model) ([]ovsdb.Operation,
 	return c.WhereAny(m, cond).Wait(ovsdb.WaitConditionNotEqual, &timeout, m, field)
 }
 
-// getAllUpdatableFields returns a list of all of the columns/fields that can be updated for a model
-func getAllUpdatableFields(model model.Model) []interface{} {
+func getCustomUpdatableFields(model model.Model, fieldNames []nbdb.LSPCustomField) []interface{} {
```

Comment:
NIT: I think this could use a doc comment

Reply:
Added